README.md

```diff
@@ -128,6 +128,7 @@ module API. Options include:
 {
   container: '#some-query-selector'; // container to attach to
   APIUrl: 'https://www.example.com/.netlify/functions/identity'; // Absolute url to endpoint.  ONLY USE IN SPECIAL CASES!
+  namePlaceholder: 'some-placeholder-for-Name'; // custom placeholder for name input form
 }
```

Comment:
@shortdiv I believe the readme is already updated with the new option but I can definitely change the example to use "nombre" instead if you want

Reply:
Ooof, I somehow missed that. 🤦🏾 Merging.